src/assets/css/global.css

```diff
@@ -0,0 +1,324 @@
+}
+
+.icon--inactive {
+    color: var(--font-color);
+}
+
+
+/* QUESTION  */
+
+
+.question-container {
+    padding: 2rem;
+    margin: 4rem 2rem 4rem 2rem;
+    text-align: justify;
+    background-color: var(--bg-question-box);
+}
+
+.question {
+    padding: 2em 4em;
+    text-align: center;
+}
+
+.question__text {
+    font-size: 4rem;
+    font-weight: 600;
+    line-height: 1;
+}
+
+
+/* ANSWERS  */
+
+.answer-container {
+    position: relative;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-around;
+    width: 80%;
+}
+
+.answer-container--blur { 
+    filter: blur(0.5em)
+}
+
+.answer {
+    width: 50%;
+    padding: 15px;
+}
+
+.answer__button {
+    width: 100%;
+    height: 100%;
+    padding: 20px;
+    border-radius: 12px;
+    border-width: 5px; 
+    border-style: solid;
+    border-color: white;
+    font-weight: bold;
+    font-size: 1.9em;
+    transition: 150ms;
+}
+
+.answer__button:hover {
+    border-width: 5px; 
+    border-style: solid; 
+    border-color: #45A29E;
+}
+
+.answer__button--correct {
+    border-width: 5px; 
+    border-style: solid; 
+    border-color: #19D415;
+}
+
+.answer__button--incorrect {
+    border-width: 5px; 
+    border-style: solid; 
+    border-color: #D41C1C;
+
+}
+
+.button {
+    background-color: var(--bg-question-box);
+    color: white;
+    width: auto;
+    padding: 0.25em 1em;
+    border-radius: 1.2em;
+    font-weight: bold;
+    transition: 150ms;
+}
+
+.button:hover {
+    box-shadow: inset -0.1em -0.5em 1em rgba(0, 0, 0, 0.25);
+}
+
+.button-skip {
+    color: black;
+    font-size: 2rem;
+}
+
+
+.button-question-read {
+    position: absolute;
+    top: 50%;
+    font-size: 3rem;
+}
+
+
+/* PATH */
+
+.choice-container {
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    align-content: space-around;
+    width: 80%;
+}
+
+.container__title {
+    font-size: 5rem;
+    text-align: center;
+    padding: 1.5rem;
+    margin-bottom: 4rem;
+}
+
+.choice {
+    width: 50%;
+    padding: 2rem;
+    justify-content: center;
+    align-items: center;
+    align-content: center;
+    display: flex;
+    flex-direction: column;
+    transition: transform 0.5s;
+}
+
+.choice:hover {
+    transform: scale(1.2);
+    color: var(--primary);
+    cursor: pointer;
+}
+
+.choice__img {
+    max-height: 100%;
+    max-width: 100%;
+    border-radius: 70%;
+    width: 27rem;
+    max-height: 27rem;
+
+}
+
+.choice__title {
+    text-align: center;
+    padding: 1.5rem;
+    font-size: 3rem;
+}
+
+@media screen and (max-width: 600px) {
+    .choice-container {
+        flex-direction: column;
+    }
+}
+
+
+/* TIER */
+
+.choice-container--tier {
+    flex-wrap: wrap;
+}
+
+
+/* NICKNAME */
+
+input {
+    width: 40rem;
+    border: 0.5rem solid var(--primary);
+    color: #000;
+    font-weight: bold;
+    padding: 1.5rem 2.5rem;
+    margin: 3rem;
+    font-size: 2em;
+    border-radius: 0.8rem;
+}
```

Comment:
We should avoid setting styles for html tags if it's not reset or normalization because it will affect every tag and can break styles written by someone else. If it is reset then it should go to the `reset.css`. Same goes for `body` and `section` at the top of this file.